Frogies-Project/Assets/Scripts/WaveSystem/DayTimer.cs

```diff
@@ -2,18 +2,20 @@
 using DG.Tweening;
 using TMPro;
 using UnityEngine;
+using UnityEngine.Serialization;
 
 namespace WaveSystem
 {
     public class DayTimer : MonoBehaviour
     {
         [SerializeField] private TMP_Text timeText;
         [SerializeField] private DayNightVisuals dayNightVisuals;
+        [SerializeField] private bool isFixed;
         
         public Action OnDayEnd;
-        public bool _isDay;
+        public bool isDay;
```

Comment:
IsDay

Reply:
Rider, said that isDay is correct as it's not a property. But I can make it as a property